cluster/gke/util.sh

```diff
@@ -181,6 +181,11 @@ function kube-up() {
 
   # Bring up the cluster.
   "${GCLOUD}" ${CMD_GROUP:-} container clusters create "${CLUSTER_NAME}" "${create_args[@]}"
+
+  if [[ ! -z "${HEAPSTER_MACHINE:-}" ]]; then
```

Comment:
@zmerlynn - why can't we do a change like that?
Then we don't need more quota, and it's pretty local change (and it seems to me that this should work for our purposes).


Obviously the change is not ready, but if you think it's worth proceeding, I can try finishing it tomorrow.